visualization/app/codeCharta/ui/matchingFilesCounter/matchingFilesCounter.component.ts

```diff
@@ -29,26 +27,25 @@ export class MatchingFilesCounterController implements NodeSearchSubscriber, Bla
 
 	public onNodeSearchComplete(searchedNodes: CodeMapNode[]) {
 		this.searchedNodeLeaves = this.getSearchedNodeLeaves(searchedNodes)
-		this.updateViewModel(this.searchedNodeLeaves, this._viewModel.blacklist)
+		this.updateViewModel(this.searchedNodeLeaves)
 	}
 
 	public onBlacklistChanged(blacklist: BlacklistItem[]) {
-		this._viewModel.blacklist = blacklist
-		this.updateViewModel(this.searchedNodeLeaves, this._viewModel.blacklist)
+		this.updateViewModel(this.searchedNodeLeaves)
 	}
 
-	private updateViewModel(searchedNodeLeaves: CodeMapNode[], blacklist: BlacklistItem[]) {
+	private updateViewModel(searchedNodeLeaves: CodeMapNode[]) {
```

Comment:
Remove parameter `searchedNodeLeaves` as it is stores inside the class and can be used from every function

Reply:
done